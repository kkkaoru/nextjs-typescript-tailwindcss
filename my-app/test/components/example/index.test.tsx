import { render } from '@testing-library/react';
import { Example } from '@/components/example';

describe('Example', () => {
  it('should be matched snapshot', () => {
    const { asFragment } = render(< Example />, {});
    expect(asFragment()).toMatchSnapshot();
  })
});
