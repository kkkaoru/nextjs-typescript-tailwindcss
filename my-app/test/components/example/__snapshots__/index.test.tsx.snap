// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example should be matched snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-col items-center justify-center w-full py-10 font-bold text-9xl after:block after:w-full after:h-1 after:bg-red-100"
  >
    Example
  </div>
</DocumentFragment>
`;
